doc/tutorial.md

```diff
@@ -4,9 +4,14 @@
 
 - [Overview](#overview)
 - [Setup and Code Example](#setup-and-code-example)
-- [Gas Estimation for Publishing a Module](#gas-estimation-for-publishing-a-module)
 - [Publishing Modules](#publishing-modules)
+- - [Estimating Gas for Module Publication](#estimating-gas-for-module-publication)
+- - [Publication](#publication)
```

Comment:
Would this fix the rendering problem?
```suggestion
  - [Estimating Gas for Module Publication](#estimating-gas-for-module-publication)
  - [Publication](#publication)
```

Reply:
What do you see? :upside_down_face: I did not see any issues...

Reply:
Done